runtime/lexing.c

```diff
@@ -130,10 +130,10 @@ static void run_mem(char *pc, value mem, value curr_pos) {
     src = *pc++ ;
     if (src == 0xff) {
       /*      fprintf(stderr,"[%hhu] <- %d\n",dst,Int_val(curr_pos)) ;*/
-      Field(mem,dst) = curr_pos ;
+      Store_field(mem,dst,curr_pos);
```

Comment:
`Field` is an l-value again. The changes may be reverted.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/a7900a91519535d3be91970f0b43d859dd98cded